### PR TITLE
Disable CopyToSwapchain pass for Mobile pipeline

### PR DESCRIPTION
**Pull request by akioCL**
## What does this PR do?

Disable extra pass CopyToSwapchain from mobile pipeline. The MobileDisplayMapper pass renders directly to the swapchain so there's no need for a copy. It's only needed if there's a need for upscaling.

## How was this PR tested?

Run PC DX12 and Vulkan. Run Android.

### CODE REVIEW

**Review by Twolewis — APPROVED** [review]

**Comment by moudgils** [comment]
Note: To not merge until iOS Is tested. I can try to find time to sync and test but my availability is very sporadic atm.

**Comment by Twolewis** [comment]
I'll make a build today to test

**Comment by Twolewis** [comment]
@akioCL - can you merge from carbonated/dev into this branch?  Since there's a revert commit affecting many of the same files, there's conflicts that I'll need your eyes on to resolve.  Once merged I'll cut an ios branch and make sure it all works.

Thanks!

**Comment by moudgils** [comment]
I did do some testing locally. Since this PR doesn't actually enable removing CopyToSwapchain pass from iOS it actually doesn't break iOS. I did enable removing the pass locally, made a bunch of fixes and do have it running. Feel free to merge this PR. I can sync to it and create a new PR that brings over this optimization for iOS.

**Review by moudgils — APPROVED** [review]

**Pull request merged**
